### PR TITLE
WEB-2817: Adding Power Zones and workout type family/location

### PR DESCRIPTION
**Pull request by JustinDunnWahoo**
This goes with https://github.com/WahooFitness/wahoo-cloud/pull/60

### CODE REVIEW

File: source/includes/_workout_types.md [hunk 1]
```diff
@@ -1,63 +1,88 @@
...
+---------- | ---------------------------- |
+0          |  BIKING                      |
+1          |  RUNNING                     |
+2          |  SWIMMING                    |
+3          |  WATER SPORTS                |
+4          |  SNOW SPORTS                 |
+5          |  SKATING                     |
+6          |  GYM                         |
+9          |  WALKING                     |
+30         |  N/A                         |
```
Comment (aengesath) [inline source/includes/_workout_types.md:25]: @JustinDunnWahoo can you check that this isn't supposed to be "UNKNOWN"?

File: source/includes/power_zones/_update.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
+## Update a Power Zone
+
+Requires the `power_zones_write` scope
+
+Updates a power zone for the authenticated user.
+
+```shell
+curl --header "Authorization: Bearer 414c6a44a5f8b918830b370db05" -X PUT
+  -d power_zone[ftp]=300 api.wahooligan.com/v1/power_zones/56519
```
Comment (aengesath) [inline source/includes/power_zones/_update.md:9]: @JustinDunnWahoo missing https:// prefix on url .. other examples have it.. also, remove the bearer token and use placeholder

File: source/includes/workout_summaries/_show.md [hunk 3]
```diff
@@ -12,18 +12,18 @@ curl --header "Authorization: Bearer users-token-goes-here" https://api.wahoolig
...
-    "power_avg": 94.59,
     "power_bike_np_last": 150,
-    "power_bike_tss_last": 304.90,
-    "speed_avg": 10.75,
+    "power_bike_tss_last": "304.90",
+    "power_avg": "94.59",
+    "speed_avg": "10.75",
     "work_accum": 104148000,
     "file": {
         "url": "https://wahoo-cloud-web.s3.amazonaws.com/development/uploads/workout_file/file/EjA4DJCoIaG-f2fB2MLLLg/4_Mile_Segment_.fit"
```
Comment (aengesath) [inline source/includes/workout_summaries/_show.md:29]: @JustinDunnWahoo replace this with a sample url that doesn't map to a real file

File: source/includes/workouts/_create.md [hunk 4]
```diff
@@ -10,7 +10,7 @@ curl --header "Authorization: Bearer 414c6a44a5f8b918830b370db05"  -X POST
       workout[workout_token]=123&
       workout[workout_type_id]=40&
       workout[starts]=2015-08-12T09:00:00.000Z&
-      workout[minutes]=12'  api.lvh.me:3000/v1/workouts
+      workout[minutes]=12'  api.wahooligan.com/v1/workouts
```
Comment (aengesath) [inline source/includes/workouts/_create.md:13]: @JustinDunnWahoo add https:// and remove bearer token.. check the rest of the site for others like this

**Review by aengesath — CHANGES_REQUESTED** [review]

File: source/includes/_workout_types.md (hunk 1 above)
Comment (JustinDunnWahoo) [reply source/includes/_workout_types.md:25]: @aengesath Unknown for Families is anything that does not have an ID according to the docs https://wahooligans.atlassian.net/wiki/spaces/AR/pages/11259510786/Wahoo+Workout+Types

but I am going to add Unknown to the workout types list, its ID is 255 on production

**Review by aengesath — APPROVED** [review]

**Pull request merged**